src/main/java/hudson/plugins/accurev/AccurevSCM.java

```diff
@@ -877,8 +890,21 @@ public boolean isUsePromoteListen() {
         public void setUsePromoteListen(boolean usePromoteListen) {
             this.usePromoteListen = usePromoteListen;
         }
+        
+      
+
+        
+       
+        public boolean isServerDisabled() {
+			return isServerDisabled;
+		}
+        
+        @DataBoundSetter
+		public void setServerDisabled(boolean isServerDisabled) {
+			this.isServerDisabled = isServerDisabled;
+		}
 
-        public boolean migrateCredentials() {
+		public boolean migrateCredentials() {
```

Comment:
Don't use tab, please install https://marketplace.eclipse.org/content/editorconfig-eclipse as it says spaces